frontend/client/__tests__/navbar.test.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import { rootStore } from '../src/store/model'
+import MenuItem from '@material-ui/core/MenuItem'
+import NavBar from '../src/components/NavBar'
+import { auth } from '../src/store/firebase'
+import { createStore } from '../src/store'
+
+beforeAll(() => {
+  rootStore.user.__update({ isSignedIn: true, isFirstTimeUser: false, firstName: 'testUser' })
+})
+
+// Mock Redirect to avoid router error
+jest.mock('react-router-dom', () => {
```

Comment:
What is this doing?

Reply:
This is mocking the Redirect component used in NavBar. If we don’t do this, an error is thrown than you can’t use Redirect outside of a Router. So, we just replace the Redirect component with an empty div